understanding/20/use-of-color.html

```diff
@@ -154,21 +154,34 @@ <h2>Benefits of Use of Color</h2>
+         <p>Students view an SVG image of a chemical compound and identify the chemical elements
                present based <strong>both</strong> on the colors used, as well as numbers next to each
                element. A legend shows the color and number for each type of element. Sighted users who
                cannot perceive all the color differences can still understand the image by relying on
-               the numbers.</dd>
-      </dl>
-      
+               the numbers.</p>
+     
+         <h3>Focus on links</h3>
+         <p>A block of text contains a non-underlined blue hyperlink that has a contrast of 3:1 to the surrounding text. When receiving keyboard focus, the link's text color changes to a darker blue. The contrast difference between the link color in the focused and the unfocused state is only 2:1. In addition, when focused the contrast difference between the link's text color and the surrounding text falls well below a 3:1 ratio. However, to further emphasize the focused state the link is provided an additional underline. Due to the additional underline, color is not used as the only visual indicator of the focused hyperlink, so it passes Success Criterion 1.4.1 Use of Color. Without the underline, the hyperlink would have failed this Success Criterion.</p>
+     <figure id="figure-text-links">
+          <img src="img/textlink.svg" alt="Two instances of the same paragraph of text containing a hyperlink. The first instance shows the unfocused hyperlink with blue colored text with no underline. The second instance shows the same hyperlink in the focused state, now with darker blue text and an underline" width="500" />
+				<figcaption>In the default state, the blue hyperlink has a contrast of 5.5:1 to the white background. When focused the hyperlink's darker blue text color has a contrast of 10.8:1 to the background. However, the difference of contrast to the link color in unfocused state is only 2.0:1 (i.e., below 3:1). </figcaption></figure>
+				</section>
+				<section>
+     
+         <h3>Focus on buttons</h3>
+         <p>A page has a row of buttons with white text on a medium blue background. When one of the buttons receives keyboard focus, its background turns black. The difference of contrast between the unfocused and the focused button background is above 3:1. This difference of contrast means that a change of color is not the only indicator of focus.</p>
+            <figure id="figure-buttons">
+          <img src="img/buttons.svg" alt="Three buttons with white text on a blue background. When a button receives keyboard focus, the background turns black" width="400" />
+				<figcaption>A row buttons that all have white text on a mid-blue background with a contrast of 5:1. The black button represents a button which has received keyboard focus. The difference of contrast between the focused and the unfocused state of the button's background color is 4.2:1.</figcaption></figure>
+     
```

Comment:
```suggestion
     
     <h3>Link state</h3>
     <p>Links on a page appear differently when a user has visited them before. Unvisited links appear blue with a solid underline, and visited links appear purple with a dashed underline. The difference in underline types means that color is not used as the only visual indicator of links' states.</p>
```

Could we add something about distinguishing link states by color? I find this is often overlooked by designers and auditors, so mentioning it explicitly would be helpful.

If it's too late in the review to add this I completely understand. If it could be added, though, I really think it would be helpful.

Reply:
PS: Not sure what the process is for suggesting non-blocking changes to commits, so I approved the whole PR to indicate that. Please let me know if I should have done it a different way.

Reply:
FWIW In the past this group has not addressed visited and unvisited state color use because of the problems of trying to get 3 colors that all contrast sufficiently with each other.  Historically folks have not flagged issues comparing visited and unvisited states because browsers have some data on that and many sites today override that information.